index.html.in

```diff
@@ -368,7 +368,7 @@
               </div>
               <div class="visible-xs">
                   <select class="localeSelect pull-left"><option> </option></select>
-                  <i class="fa fa-globe fa-2x pull-left localeGlobe" style="padding: 5px 0px 0px 5px"></i>
+                  <i class="fa fa-globe fa-2x pull-left localeGlobe" id="localeGlobeIconXs"></i>
```

Comment:
Do you mind switching this to `XS` instead of `Xs`? The latter looks really strange...